### ISSUE TITLE
Look at binary packaging solutions

### ISSUE THREAD

**Issue by windymilla** [DistributedProofreaders/guiguts-py#29]
Consider pyinstaller et al and see what works on a Mac with its digital signing, etc.

**Comment by tangledhelix** [comment]
I already paid for an Apple Developer account so I'll volunteer to test anything involving signing. And if it works, to maintain the Apple packaging.

**Comment by tangledhelix** [comment]
Just a note to mention #274 will end up intertwining with this.

**Comment by windymilla** [comment]
Given our current installation method is relatively simple (install python then `pip install guiguts`) what are people's thoughts on how urgently we need to do this? I suppose whether it needs to be before or after the first beta release (when we will claim that GG2 is suitable to use for the whole PPing task).

**Comment by srjfoo** [comment]
Given the hoops users have to jump through to install GG1, still, I don't think that it's unreasonable to ask them to do the python install (if they don't have it) and use `pip install guiguts` well into beta (depending on how many beta releases there are).

**Comment by tangledhelix** [comment]
IMO it’s not urgent for beta phase really. As @srjfoo points out, GG1 release is complex enough that the pip approach isn’t a higher bar anyway.

**Comment by tangledhelix** [comment]
Should this get split into 2-3 issues, i.e. per platform? The Win vs. Mac installation will be completely different and I was considering looking into Ubuntu too. But tracking all that in one issue might not make sense.

**Comment by windymilla** [comment]
@tangledhelix - if you are planning to work on it in the near future for one particular platform, please feel free to enter a separate issue to cover the work you intend to do. If you're still of the opinion that it's not urgent for the beta phase, and that the pip approach is OK for that, then we can just keep this one issue to cover all platforms for now.

**Comment by windymilla** [comment]
@tangledhelix - do you think we should/could change this to "future feature" instead of "core"?

**Comment by tangledhelix** [comment]
@windymilla Yes, I think so. The way things are shaping up with the pip install, it doesn't seem very urgent.

**Comment by srjfoo** [comment]
I agree.

**Comment by windymilla** [comment]
As per Slack discussion, closing this. We can always resurrect the idea if needed later.
